test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -280,7 +280,7 @@ std::string RandomCodeBase::generate(int _maxOpNumber, RandomCodeOptions const&
 			opcode = makeOpcodeDefined(opcode);
 			eth::Instruction inst = (eth::Instruction) opcode;
 			eth::InstructionInfo info = eth::instructionInfo(inst);
-			if (info.name.find("PUSH") != std::string::npos)
+			if (strcmp(info.name, "PUSH") == 0)
```

Comment:
It's not the same. The `.find()` searches for a substring, so it will match all `PUSHxx` instructions.
Better leave it as it was:
```
if (std::string{info.name}.find("PUSH") != std::string::npos)
```

Reply:
@chfast  Ah, thank you.  My apologies for not fully understanding the difference.  I'll make that change.